presto-docs/src/main/sphinx/overview/concepts.rst

```diff
@@ -183,6 +183,33 @@ that statement. A query encompasses stages, tasks, splits, connectors,
 and other components and data sources working in concert to produce a
 result.
 
+Query Plan
+^^^^^^^^^^
+
+A query plan is an `operator <https://prestodb.io/docs/current/overview/concepts.html#operator>`_ 
+- a sequence of steps used to access and manipulate data according to the SQL 
+query. It is represented as a tree of nodes, with each node representing a 
+logical unit of relational algebra that can be executed by the engine. 
+Because query plans can have different performance behavior, Presto uses 
+a query optimizer to make query plans more efficient.
```

Comment:
I think we should change this to -

> Since SQL is declarative, multiple query plans can be generated to execute a given query. Because query plans can have different performance behavior, Presto uses a query optimizer to choose an efficient plan.